coolwsd.xml.in

```diff
@@ -184,7 +184,7 @@
         <host desc="Ditto, but as IPv4-mapped IPv6 addresses">::ffff:10\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}</host>
         <host desc="Localhost access by name">localhost</host>
       </lok_allow>
-      <content_security_policy desc="Customize the CSP header by specifying one or more policy-directive, separated by semicolons. See w3.org/TR/CSP2"></content_security_policy>
+      <content_security_policy desc="Customize the CSP header by specifying one or more policy-directive, separated by semicolons. See https://www.w3.org/TR/CSP2/#directive-frame-ancestors"></content_security_policy>
```

Comment:
it allow more than just frame ancestors. Maybe it the next line should include a link to frame-ancestors CSP part of the deprecation.